analyzers/tests/SonarAnalyzer.UnitTest/Helpers/BlockIdMapTest.cs

```diff
@@ -1,23 +1,22 @@
-﻿extern alias csharp;
```

Comment:
I don't know why this was being used in the first place, but it's great to see it gone 🎉